Valet/VALSecureEnclaveValet.m

```diff
@@ -80,6 +80,18 @@ + (BOOL)_macOSElCapitanOrLater;
 #endif
 }
 
++ (BOOL)_macOSSierraOrLater;
+{
+#if TARGET_OS_MAC && __MAC_10_12
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wtautological-compare"
+    return (&kSecAttrTokenIDSecureEnclave != NULL);
```

Comment:
Because you may have built against the 10.12 SDK but you're running on 10.11. The compile-time check is SDK rather than OS